packages/kit/src/runtime/app/forms.js

```diff
@@ -14,12 +14,26 @@ export function deserialize(result) {
 	return parsed;
 }
 
+/**
+ * @param {string} oldName
+ * @param {string} newName
+ * @param {string} callLocation
+ * @returns void
+ */
+function warnOnAccess(oldName, newName, callLocation) {
```

Comment:
1. Can we call this `warn_on_access` for snake case consistency?
2. Can we remove the `if (!DEV)` check from here and instead only customize these getters in dev mode? If Terser/esbuild is smart enough to remove calls to this whole function when it's a no-op, then maybe this isn't so important.

Reply:
lol if I had a penny for every time I forgot snake case...

Reply:
The no-op function call _is_ removed altogether in prod, FWIW. The overhead we're left with is this, but I don't think there's a way to get rid of it without adding more code elsewhere

```diff
-data: form_data,
+get data() { return form_data },
```